src/redux/store.jsx

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import cartReducer from './Books/bookSlice';
+
+const store = configureStore({
+  reducer: {
+    book: cartReducer,
+  },
+});
```

Comment:
Kindly also import the categoriesSlice also and add it to the reducer list inside the store.